scripts/publish-release.sh

```diff
@@ -4,20 +4,20 @@
 set -ex
 
 ## Obtain/check npm access
-npm whoami
-if [ "$?" != "0" ];
+if ! npm whoami;
   then
     npm login
 fi
 node ./scripts/npm-access.js
 
 ## Publish packages to npm
 lerna version
-lerna publish from-package 
+lerna publish from-package
 
 ## Update git branches
+RELEASE_BRANCH_NAME=$(git branch --show-current)
```

Comment:
Hm, I was going to say "this bothers me because it depends on people being on the correct branch", but now that I look I guess that was actually already true, so, uh, I guess that's probably not a problem?

If you do want to do something about this, you could alter the release branch to just be based on the current day rather than time, that way one could switch to it reliably (well, unless you're doing the release around midnight).  I dunno, probably this isn't worth worrying about, and the existing code already has this caveat, but I thought I'd point it out at least.

Reply:
Yeah, but I also wanted to ensure that the branch would have a unique name. I vote we don't worry about this at this time. Let me know if you decide to disagree :)